packages/editor/src/lib/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default as JSONEditor } from './json-editor.svelte';
+export { default as JSONDiff } from './json-diff.svelte';
```

Comment:
```suggestion
export { default as JsonEditor } from './json-editor.svelte';
export { default as JsonDiff } from './json-diff.svelte';
```

Opinions?

Reply:
I like the suggested change - that's the casing that auto-imports use.

Reply:
Migrated some of the usage of JSON to Json in 53ef158a